CONTRIBUTING.md

```diff
@@ -113,84 +117,71 @@ It is a good idea to create a detailed (but concise) comment to accompany the in
 * Any questions or other comments for contribution reviewers
 
 #### Review Process
-All pull requests will approved by a sufficient number of project maintainers (those with write access to the project repository) before any changes are merged. Before aproving changes, reviewers will determine whether:
+All pull requests will need to be approved by a sufficient number of maintainers before any changes are merged. Before approving changes, reviewers will make sure that:
 * Changes do not interfere with existing features or design plans
-* There are overlap issues (i.e. other developers are already assigned to it) 
+* There are no overlap issues (i.e. other developers are already assigned to it) 
 * Sufficient testing occurred
 * Quality standards are upheld
 
-While it is certainly _possible_ that a quality pull request might be accepted right away, it should be expected that there will be some changes requested of the pull request, and issues that need to be discussed and resolved before it is approved. The pull request post itself is a great place to have these conversations, though contributors should not be afraid to directly message fellow developers either, when doing so is helpful to resolving any issues.
+While it is certainly _possible_ that a quality pull request might be accepted right away, change requests, questions, and discussions should be expected before the pull request can be merged. Discussions around the pull request should preferable be held in the pull request comments. This way it is easy for multiple people to follow progress... but don't hesitate to reach out directly to the developers and maintainers either!
+
+
+it should be expected that there will be some changes requested of the pull request, and issues that need to be discussed and resolved before it is approved. The pull request post itself is a great place to have these conversations, though contributors should not be afraid to directly message fellow developers either, when doing so is helpful to resolving any issues.

```

Comment:
Isn't this just saying the same as what's written above?
```suggestion
```

Reply:
Yup! Looks like the old phrasing (or maybe an edit in progress I forgot to delete) somehow survived. Should be dead after this commit though. For real this time!